Interface/FbMessage.cs

```diff
@@ -0,0 +1,181 @@
+﻿using OscCore;
+using System;
+
+namespace QuestProModule;
+
+public class FbMessage
+{
+  private const int NaturalExpressionsCount = 63;
+  private const float SranipalNormalizer = 0.75f;
+  public readonly float[] Expressions = new float[NaturalExpressionsCount + 8 * 2];
+
+  public void ParseOsc(OscMessageRaw message)
```

Comment:
these messages should probably go into AlvrConnection, as these are called in the connection's side of the pond.